libnsh.config

```diff
@@ -0,0 +1,5 @@
+BR2_PACKAGE_LIBNSH=y
```

Comment:
This file should not be added to NetBox (or even Buildroot), you probably used this for testing the package, so probably a mistake anyway.